config/management/src/genesis.rs

```diff
@@ -33,7 +33,8 @@ impl Genesis {
         let genesis = vm_genesis::encode_genesis_transaction_with_validator(
             association_key,
             &validators,
-            Some(libra_types::on_chain_config::VMPublishingOption::open()),
+            Some(libra_types::on_chain_config::VMPublishingOption::open()), // TODO: intended?
+            ChainId::test(), // TODO: where should we read this from?
```

Comment:
@JoshLind @davidiw: I am happy to fix this if it's easy or leave the TODO if not... Also, wondering  about the line  above.

Reply:
I think we'll want to add this to the layout. Feel free to leave it as a todo and we can jump on it, since the layout has other tweaks that need to be made.